scripts/up.sh

```diff
@@ -1,79 +1,61 @@
 #!/usr/bin/env bash
-#
-# Bring up the SDK test environment.
-
-set -e
-
-rootdir=`dirname $0`
-pushd $rootdir/.. > /dev/null
-
-# Defaults
-TYPE_OVERRIDE=""
-DAEMON_FLAG="-d"
-SKIP_BUILD=0
-SHOW_HELP=0
-ENV_FILE=".up-env"
-PARALLEL="--parallel"
-
-show_help() {
-  echo "Manage bringing up the SDK test environment."
-  echo
-  echo "Usage: up.sh [options]"
-  echo
-  echo "Options:"
-  echo "  -f <FILE>  Override the environment file."
-  echo "  -t <TYPE>  Override the installation type specified in the environment file."
-  echo "             Valid types: ['channel', 'type']"
-  echo "  -s         Skip rebuilding the docker image."
-  echo "  -i         Start the docker environment in interactive mode."
-  echo "  -h         Provide this help information."
-  echo "  -p         Disable parallel build mode. Probably only useful when editing the test environment."
-}
-
-# Parse arguments
-while getopts "pf:t:sih" opt; do
-  case "$opt" in
-    f) ENV_FILE=$OPTARG; ;;
-    i) unset DAEMON_FLAG; ;;
-    s) SKIP_BUILD=1; ;;
-    t) TYPE_OVERRIDE=$OPTARG; ;;
-    h) show_help; exit 0 ;;
-    p) unset PARALLEL; ;; 
-  esac
-done
-
-# Verify there are no positional parameters with getopt/getopts
-shift "$((OPTIND-1))"
-if [[ "$1" != "" ]]; then
-  echo "No positional arguments should be provided, found '$@'"
-  echo
-  show_help
-  exit
-fi
 
-# Load environment.
-source $ENV_FILE
+set -euo pipefail
+
+START=$(date "+%s")
+
+THIS=$(basename "$0")
+ENV_FILE=".env"
+echo "$THIS: sourcing environment vars from-->$ENV_FILE"
```

Comment:
@tzaffi Optional - I'd consider printing the absolute path to `$ENV_FILE` to aid debugging.

For example, locally I enabled config overwrite and made no other change in order to see the error message.

```
❯ make harness
./test-harness.sh
~/dev/py-algorand-sdk ~/dev/py-algorand-sdk
~/dev/py-algorand-sdk/test-harness ~/dev/py-algorand-sdk ~/dev/py-algorand-sdk
down.sh: sourcing environment vars from-->.env
down.sh: looking to clean up inside-->
down.sh: directory  does not exist - NOOP
~/dev/py-algorand-sdk ~/dev/py-algorand-sdk
Cloning into 'test-harness'...
remote: Enumerating objects: 254, done.
remote: Counting objects: 100% (254/254), done.
remote: Compressing objects: 100% (218/218), done.
remote: Total 254 (delta 42), reused 149 (delta 19), pack-reused 0
Receiving objects: 100% (254/254), 4.70 MiB | 10.05 MiB/s, done.
Resolving deltas: 100% (42/42), done.
test-harness.sh: OVERWRITE downloaded test-harness/.env with .test-env:
test-harness.sh: seconds it took to get to end of cloning and copying: 1s
~/dev/py-algorand-sdk/test-harness ~/dev/py-algorand-sdk ~/dev/py-algorand-sdk
up.sh: sourcing environment vars from-->.env
./scripts/up.sh: line 15: TYPE: unbound variable
make: *** [harness] Error 1
```

When faced with the shown output, I think it'd accelerate debugging to enable a direct file path copy+paste to inspect `.env`.  

If you prefer as is, feel welcomed to resolve.